MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [Recep Burak Otman]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [Popularity of Cryptos] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
```

Comment:
- [ ] Some *[APP TYPE]* text is still left in your license like the previous reviewer mentioned. Kindly customize them as this would make your documentation more professional